frontend/packages/kubevirt-plugin/src/components/modals/create-vm-wizard.tsx

```diff
@@ -21,9 +23,21 @@ import {
   VirtualMachineModel,
   NetworkAttachmentDefinitionModel,
   DataVolumeModel,
+  V2VVMwareModel,
 } from '../../models';
 import { createModalResourceLauncher } from './modal-resource-launcher';
 
+const mapStateToProps = ({ k8s }) => ({
+  kindsInFlight: k8s.getIn(['RESOURCES', 'inFlight']),
+  k8sModels: k8s.getIn(['RESOURCES', 'models']),
+});
+
+const CreateVmWizardDecorated = connect(mapStateToProps)((props) => {
+  const { kindsInFlight, k8sModels } = props;
+  const isV2vVmwareCrd = !kindsInFlight && !!k8sModels.get(V2VVMwareModel.kind);
```

Comment:
it would look nicer to resolve the flag `isV2vVmwareCrd` inside the `mapStateToProps`

Reply:
done